examples/erc20/lib.rs

```diff
@@ -59,20 +61,19 @@ mod erc20 {
         /// Creates a new ERC-20 contract with the specified initial supply.
         #[ink(constructor)]
         pub fn new(initial_supply: Balance) -> Self {
+            ink_lang::codegen::initialize_contract(|contract| Self::new_init(contract, initial_supply))
```

Comment:
@Robbepop using the `codegen` module here feels wrong, is there a better way to expose this to users?

Reply:
The actual plan is still that we provide some useful and well designed syntactic sugar so that this line of code won't be necessary but maybe that's not well thought through and maybe writing out this line of code actually isn't so bad. Then we could put this into some other module or into root.